packages/admin-panel/src/importExport/ImportModal.js

```diff
@@ -161,8 +157,11 @@ export const ImportModalComponent = React.memo(
               errorMessage={fileErrorMessage}
               isLoading={status === STATUS.LOADING}
             >
-              {status === STATUS.SUCCESS ? (
-                <p>{successMessage}</p>
+              {status === STATUS.TIMEOUT ? (
+                <p>
+                  Import is taking a while, and will continue in the background. You will be emailed
+                  when the import process completes.
```

Comment:
Decided the message being displayed here was more of a front end concern, the back end should really just return the status (makes generic handling slightly easier on the back end too)